src/Tests/InstrEngineTests/InstrEngineTests/PathUtils.cs

```diff
@@ -14,14 +14,18 @@ public static class PathUtils
         private const string ResourcesFolder = "ExtractedResources";
 
         // All paths are relative to the AnyCPU binaries directory e.g. bin\Debug\AnyCPU
-        public const string BaselinesBinPath = @".\" + BaselinesFolder;
-        public const string TestScriptsBinPath = @".\" + TestScriptsFolder;
-        public const string InstrumentationEngineX64BinPath = @"..\..\x64\MicrosoftInstrumentationEngine_x64.dll";
-        public const string InstrumentationEngineX86BinPath = @"..\..\x86\MicrosoftInstrumentationEngine_x86.dll";
-        public const string NaglerInstrumentationConfigX64BinPath = @"..\..\x64\NaglerInstrumentationMethod_x64.xml";
-        public const string NaglerInstrumentationConfigX86BinPath = @"..\..\x86\NaglerInstrumentationMethod_x86.xml";
-        public const string NaglerInstrumentationMethodX64BinPath = @"..\..\x64\NaglerInstrumentationMethod_x64.dll";
-        public const string NaglerInstrumentationMethodX86BinPath = @"..\..\x86\NaglerInstrumentationMethod_x86.dll";
+        // Use forward slashes '/' to maintain compatibility between Windows and Linux
+        public const string BaselinesBinPath = @"./" + BaselinesFolder;
+        public const string TestScriptsBinPath = @"./" + TestScriptsFolder;
+        public const string InstrumentationEngineX64BinPath = @"../../x64/MicrosoftInstrumentationEngine_x64.dll";
+        public const string InstrumentationEngineX86BinPath = @"../../x86/MicrosoftInstrumentationEngine_x86.dll";
+        public const string NaglerInstrumentationConfigX64BinPath = @"../../x64/NaglerInstrumentationMethod_x64.xml";
+        public const string NaglerInstrumentationConfigX86BinPath = @"../../x86/NaglerInstrumentationMethod_x86.xml";
+        public const string NaglerInstrumentationMethodX64BinPath = @"../../x64/NaglerInstrumentationMethod_x64.dll";
+        public const string NaglerInstrumentationMethodX86BinPath = @"../../x86/NaglerInstrumentationMethod_x86.dll";
+        public const string LinuxInstrumentationEngineX64BinPath = @"../../../../out/Linux/bin/x64.Debug/ClrInstrumentationEngine/libInstrumentationEngine.so";
```

Comment:
>out/Linux/bin/x64.Debug [](http://example.com/codeflow?start=81&length=23)

These aren't available during build

Reply:
I wish unit tests supported conditionally ignoring tests. Maybe separate the tests into categories, and only run the Windows tests on Windows, and the other tests on other platforms?

Reply:
Yeah, that's essentially the same thing. You can filter on categories as well. Something like this:
```
internal abstract class TestClassImpl
{
    [TestMethod]
    void Method1() {...}
}

[TestClass]
[Category("Windows")];
[DeploymentItem("WindowsFile", ...)]
internal class WindowsTest : TestClassImpl {}

[TestClass]
[Category("Linux")];
[DeploymentItem("LinuxFile", ...)]
internal class LinuxTest: TestClassImpl {}
```